libs/@blockprotocol/graph/src/temporal/main.ts

```diff
@@ -3,7 +3,7 @@
  * This defines the main types and type-guards used when working with the Graph module.
  */
 
-import { BaseUri } from "@blockprotocol/type-system/slim";
+import {BaseUri} from "@blockprotocol/type-system/slim";
```

Comment:
This is probably a formatting error

Reply:
Hm, I wonder why it wasn't picked up by my IDE

Reply:
Seems like prettier had become badly configured. Fixed it, cheers